simpledotorg/simple-server
# Return remaining account lockout duration

**Story card:** [sc5042](URL)

## Because
 Throws 500 internal server error when :
- user tries to login with an unknown phone number
- soft deleted user tries to login with his/her phone number

## This addresses

Throw 401 with proper error message instead of 500 internal server error when :
- A user tries to login with an unknown phone number
- A soft deleted user tries to login with his phone number

## Test instructions

- Add test for checking if login controller throws 401 with proper message when a user tries to login using unknown phone number
- All test should pass
